lib/components/Progress.vue

```diff
@@ -1,40 +1,44 @@
 <template>
     <div class="progress">
-        <div ref="myBar" class="progressBar">
-            {{ progressValue }}
-        </div>
+        <div ref="myBar" class="progressBar"></div>
     </div>
 </template>
 
 <script>
 export default {
     name: 'Progress',
-    props: [
-        'value',
-        'valueChange',
-        'maxValue',
-    ],
-    data () {
-        return {
-            id: setInterval(this.frame, 50),
-            progressValue: this.value,
-        };
+    props: {
+        value: {
+            type: Number,
+            default: 0
+        },
+        maxValue: {
+            type: Number,
+            default: 100
+        }
+    },
+    mounted(){
+        setTimeout(this.render, 10);
```

Comment:
why do we need timeout here? I also don't think we need a render method. Two computed properties should be enough.

Reply:
Added timeout for animated loading on mounted. Do i remove animation?

Reply:
we should avoid timeouts and timers as much as possible. For animations, css transitions should be our first choice. If not, then we should consider using vue transitions/animations

https://vuejs.org/v2/guide/transitions.html